packages/embark-library-manager/src/npm.js

```diff
@@ -41,28 +70,37 @@ class Npm {
         this._logger,
         packageName,
         version,
-        'Downloading and installing {{packageName}} {{version}}...',
-        'Still downloading and installing {{packageName}} {{version}}... ({{duration}})',
-        'Finished downloading and installing {{packageName}} {{version}} in {{duration}}',
+        'Downloading and installing {{packageName}} {{version}}...\n',
+        'Still downloading and installing {{packageName}} {{version}}... ({{duration}})\n',
+        'Finished downloading and installing {{packageName}} {{version}} in {{duration}}\n',
         {
           showSpinner: !this._useDashboard,
           interval: this._useDashboard ? 2000 : 1000,
-          longRunningThreshold: 10000
+          longRunningThreshold: 600000
```

Comment:
Just for clarity: Why does this threshold need to be increased?

Reply:
I bumped it when first dealing with some race conditions, but it actually doesn't have any effect on that (just determines when the text to the right of the spinner turns red). And then I forgot to change it back.

Actually, I think it should be increased from 10 seconds to 2 minutes since the `web3` package does take a little while to install and switching to red text pretty quickly might cause users to think something is going wrong with the install.

Reply:
Changes made.